rimage/image_file.go

```diff
@@ -262,11 +268,26 @@ func DecodeImage(ctx context.Context, imgBytes []byte, mimeType string) (image.I
 	if returnLazy {
 		return NewLazyEncodedImage(imgBytes, mimeType), nil
 	}
-	img, _, err := image.Decode(bytes.NewReader(imgBytes))
-	if err != nil {
-		return nil, err
+	switch mimeType {
+	case "":
```

Comment:
Think you can/should combine these by doing...  case "", ut.MimeTypeJPEG :

Reply:
done